apps/emqx_management/src/emqx_mgmt_api_clients.erl

```diff
@@ -647,7 +647,7 @@ set_keepalive(put, #{bindings := #{clientid := ClientID}, body := Body}) ->
         error ->
             {400, 'BAD_REQUEST', "Interval Not Found"};
         {ok, Interval} ->
-            case emqx_mgmt:set_keepalive(emqx_mgmt_util:urldecode(ClientID), Interval) of
```

Comment:
Q: is this change intended for this PR?  Seems unrelated.  :thinking:

Reply:
Yes, It's not related, I just fix this incidentally, because `keepalive` api is hidden for swagger.